README.md

```diff
@@ -16,20 +16,21 @@ You can add Fluent Kafka Streams Tests via Maven Central.
 
 #### Gradle
 ```gradle
-compile group: 'com.bakdata', name: 'fluent-kafka-streams-tests', version: '1.1.0'
+compile group: 'com.bakdata', name: 'fluent-kafka-streams-tests-junit5', version: '2.0.0'
 ```
 
 #### Maven
 ```xml
 <dependency>
     <groupId>com.bakdata</groupId>
-    <artifactId>fluent-kafka-streams-tests</artifactId>
-    <version>1.1.0</version>
+    <artifactId>fluent-kafka-streams-tests-junit5</artifactId>
+    <version>2.0.0</version>
 </dependency>
 ```
 
-For other build tools or versions, refer to the [latest version in MvnRepository](https://mvnrepository.com/artifact/com.bakdata.fluent-kafka-streams-tests/fluent-kafka-streams-tests/latest).
+There is also a junit4 version and one without any dependencies to a specific testing framework.
 
+For other build tools or versions, refer to the [latest version in MvnRepository](https://mvnrepository.com/artifact/com.bakdata.fluent-kafka-streams-tests/latest).
```

Comment:
This link gives me 404. Not sure if this will change once the pipeline is green? Otherwise please point to correct endpoint.

Reply:
Updated link